### DISCUSSION TITLE
Change info in About tab

### DISCUSSION THREAD

**Discussion by trungle14** [MrGreensWorkshop/MrGreen-JekyllTheme#213] [How to] [answered]
Hi Mr. Green, 
Thank you for sharing the template I did try my best to modify the content in the "About" tab but unfortunately I cannot find any file related to that to make change, could you also specifically which path I should look at?

**Comment by MrGreensWorkshop** [comment, marked as answer]
These links may be of assistance to you in the future.

 - Watching [Mr. Green theme tutorials playlist](https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD) on YouTube.
 - Searching the existing [Issues](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/issues/).
 - Checking the existing [Discussions](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/discussions/).
 - Checking the [jekyll docs](https://jekyllrb.com/docs/).
